react/README.md

```diff
@@ -34,12 +34,28 @@ For example, the following works:
 <MaskedTextInput
   mask={['(', /[1-9]/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]}
   className="form-control"
-  placeholder="Enter zip code"
+  placeholder="Enter a phone number"
   guide={false}
   id="my-input-id"
 />
 ```
 
+You can additionally use your own custom input component by passing in the prop `inputElement`.
```

Comment:
I'm not sure about the wording here. How about...

> You can alternatively pass in your own input element as the `inputElement` prop

...?

What gets passed has to be a ref to an HTML input element right?